CMSgov/design-system
# [WNMGDS-977] Table typescript conversion

## Summary
Convert Table component to typescript

### Changed
- Update the Table component and test files to typescript.
- Remove the `<div>` wrapper from root table element for non-scrollable table. The `<div>` wrapper is needed for scrollable table only

## How to test
- Check http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-922/stackable-table-updates to confirm table component works as expected
